packages/web/src/components/comments/CommentBlock.tsx

```diff
@@ -68,10 +77,14 @@ export const CommentBlock = (props: CommentBlockProps) => {
 
   return (
     <Flex w='100%' gap='l' css={{ opacity: isDeleting ? 0.5 : 1 }}>
-      <Avatar
-        css={{ width: 40, height: 40, flexShrink: 0 }}
-        src={profileImage}
-      />
+      <Box css={{ flexShrink: 0 }}>
+        <UserLink userId={commentUserId} disabled={isDeleting} popover noText>
+          <Avatar
```

Comment:
Do we not have user avatar?

Reply:
Hmm I dont see one; would be a good add tho! I'll toss that in

Reply:
wait nvm we do